.github/workflows/ci.yml

```diff
@@ -73,7 +81,9 @@ jobs:
         ./run-qa-checks
     - name: Tests
       run: |
-        coverage run --source=openwisp_controller runtests.py
+        coverage run --parallel-mode --source=openwisp_controller runtests.py
+        POSTGRESQL=1 coverage run --parallel-mode --source=openwisp_controller runtests.py
+        coverage combine
```

Comment:
Please try combining these changes with the changes in https://github.com/openwisp/openwisp-users/pull/340 (move flags to rc file and run tests in parallel), in the second call to `coverage`, in order to avoid increasing the build time excessively, we could run only the tests of the subdivision app, in this case `--keepdb` should help speed up things a bit.

```
POSTGRESQL=1 coverage run /tests/manage.py test openwisp_controller.subnet_division.tests
```

It should work!

Reply:
When using `POSTGRESQL=1` only a subset of tests are executed. The abstraction is done in `runtests.py` to allow running the tests locally as well. 

https://github.com/openwisp/openwisp-controller/blob/d25b16cf9b3271d8ee806e344423e0f01645dd2b/runtests.py#L25-L26